test/core/client_channel/resolvers/dns_resolver_test.cc

```diff
@@ -41,45 +43,51 @@
 #include "src/core/lib/uri/uri_parser.h"
 #include "test/core/util/test_config.h"
 
-static std::shared_ptr<grpc_core::WorkSerializer>* g_work_serializer;
+namespace grpc {
```

Comment:
Reverted, I thought using
```
namespace grpc {
namespace testing {
namespace {
```
was our convention for tests, a quick codesearch tells otherwise:
https://source.corp.google.com/search?q=file:third_party%2Fgrpc%20file:.*_test%5C.cc%20-%22namespace%20testing%22&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental and it's perfectly fine to use static for file local functions and variables.